### DISCUSSION TITLE
VoIP Mobile App

### DISCUSSION THREAD

**Discussion by getsean** [0perationPrivacy/VoIP#162] [General]
Just curious on the timeline for the android app? No rush just wondering if this is a 2022 thing or later next year? Thanks!

**Comment by 0perationPrivacy** [comment]
The app is almost done but it has many bugs. I didn't want to release it yet as I would get a ton of people complaining. 
The notifications and sms refreshing is causing a real issue with the lack of google or Apple push services on a GrapheneOS! That's what's holding it up now. The programmer is troubleshooting it. 
Aim is to get it out in Dec 2022 to the public on f-droid and GitHub as an apk.

**Reply by getsean** [reply]
Awesome thanks. Let me know if you need any beta testers that won't complain. 👍

**Comment by roundupglorify** [comment]
Are there any updates on this?

**Reply by 0perationPrivacy** [reply]
Yes, in the final phases of coding the custom push notifications for sms. Will release the beta on F-Droid in a week or two. (it won't have the calling feature yet)

**Reply by OfficialGitCub** [reply]
The community is really looking forward to this. Much appreciated 0P

**Comment by roundupglorify** [comment]
The app has been released [here](https://github.com/0perationPrivacy/VoIPSuite-Mobile). Thank you so much 0peration Privacy!
